### DISCUSSION TITLE
HTTP header 'Referrer-Policy' is not supported

### DISCUSSION THREAD

**Discussion by InfoDub** [matteocorti/check_ssl_cert#490] [General]
Hello,

I am trying to monitor ssl certificate of my website and getting below alert (domain name changed) - 

Status Information:	SSL_CERT CRITICAL www.example.com:443: HTTP header 'Referrer-Policy' is not supported.

This is the command used - 
check_ssl_cert --all -t 180 -H $ARG1$ -w 30

Could you please suggest what is the cause of this alert and how can that be solved. Let me know if more details required.

Thanks!

**Comment by matteocorti** [comment]
Hi,

You are specifying ```--all``` which includes ``` --check-http-headers```.

Best practices recommend the usage of a [Referrer Policy](https://www.w3.org/TR/referrer-policy/).

Something like:

```
referrer-policy: same-origin
```

**Comment by InfoDub** [comment]
Thank you for your quick response. The policy used by website is one of the supported from the list -

Referrer Policy  origin-when-cross-origin

![image](https://github.com/matteocorti/check_ssl_cert/assets/16345943/c5c1a9a0-b4ed-4676-885a-a394a48be8fc)

Reference: https://www.w3.org/TR/referrer-policy/#referrer-policy-origin-when-cross-origin

Why am I still getting the error that policy is not supported. Is there anything wrong in my monitoring command or any additional option needs to be added?

Thank you!
